sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -101,9 +101,7 @@
 
 	@Override
 	public List<FeatureConfigurationDto> getAllAfter(Date date) {
-
-		User user = userService.getCurrentUser();
-		return service.getAllAfter(date, user).stream().map(FeatureConfigurationFacadeEjb::toDto).collect(Collectors.toList());
+		return service.getAllAfter(date).stream().map(FeatureConfigurationFacadeEjb::toDto).collect(Collectors.toList());
```

Comment:
Not certain but I think this class could extend `AbstractBaseEjb` and we could save a couple of methods here.